x-pack/plugin/data-streams/qa/rest/src/test/resources/rest-api-spec/test/data-streams/120_data_streams_stats.yml

```diff
@@ -47,13 +47,17 @@ setup:
         name: simple-data-stream1
   - is_true: acknowledged
 
+  - do:
+      indices.rollover:
+        alias: "simple-data-stream1"
+
```

Comment:
Is this better to test in the "Rolled over data stream" test case?

Reply:
That test already has a test for this behavior, I'm curious how that didn't fail prior to this, maybe something about the indices being empty triggered it?